models/gc_wsi_bgseg/meta.json

```diff
@@ -0,0 +1,156 @@
+{
+  "id": "bb083d5e-b5ed-481a-8a24-00e95d301067",
+  "name": "gc_wsi_bgseg",
+  "title": "Tissue-Background segmentation in histopathological whole-slide images",
+  "summary": {
+    "description": "This algorithm segments the background and tissue in histopathological whole-slide images.",
+    "inputs": [
+      {
+        "label": "Whole-slide image",
+        "description": "Whole-slide histopathology image",
+        "format": "DICOM",
+        "modality": "SM",
+        "bodypartexamined": "WHOLEBODY",
+        "slicethickness": "",
+        "non-contrast": false,
+        "contrast": false
+      }
+    ],
+    "outputs": [
+      {
+        "type": "Prediction",
+        "valueType": "Segmentation mask",
+        "label": "WSI background and tissue segmentation mask",
+        "description": "WSI background and tissue segmentation mask (MHA). The labels are: 0-background 1-tissue. The output will have a pixel spacing",
+        "classes": []
+      }
+    ],
+    "model": {
+      "architecture": "Fully convolutional network with 7 layers, ReLU activation functions in the first 6 convolutional layers and softmax in the last one. Max pooling was inserted after each of the first 3 convolutional layers.",
+      "training": "supervised",
+      "cmpapproach": "2D"
+    },
+    "data": {
+      "training": {
+        "vol_samples": 70
+      },
+      "evaluation": {
+        "vol_samples": 38
+      },
+      "public": false,
+      "external": false
+    }
+  },
+  "details": {
+    "name": "Tissue-Background Segmentation in Histopathological Whole-Slide Images",
+    "version": "1.0.0",
+    "devteam": "Peter Bándi",
+    "type": "Segmentation",
+    "date": {
+      "weights": "2019-12-13",
+      "code": "2024-05-16",
```

Comment:
How is the code 5 years older than the weights?

Reply:
If you look closely it is the other way around: the weights are 5 years older than the code. I.e. the model is already 5 years old, but I have recently updated the codebase (minor changes).

Reply:
Still used by the community according to Sil